components/prism-elixir.js

```diff
@@ -1,4 +1,7 @@
 Prism.languages.elixir = {
+	'doc': {
```

Comment:
I agree with [the general direction you want to take](https://github.com/PrismJS/prism/pull/2773#issuecomment-784707821). However, I think that we still need to highlight `@doc|@moduledoc` and the string value. The easiest way to do that is an `inside` grammar like this:

```js
inside: {
  'attribute': /^@\w+/,
  'string': /['"][\s\S]+/
}
``` 

Thoughts?